### PR TITLE
Enh(doc) add discovery rule for windows NRPE + translation FR

### PR DESCRIPTION
**Pull request by cedricmeschin**
## Description

Adding NRPE discovery rule for disks on windows + translation FR
resumption of the new format of the PR https://github.com/centreon/centreon-documentation/pull/1220

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] Cloud (staging)
- [X] Plugin Packs (staging)
- [ ] 22.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md [hunk 1]
```diff
@@ -160,6 +191,6 @@ yum install centreon-pack-operatingsystems-windows-nsclient-05-nrpe
 | X         | NRPETIMEOUT      | 30                        | Timeout value (Default: '30')                                    |
 | X         | NRPEEXTRAOPTIONS | **-u -2 -P 8192**         | Extraoptions to use with the NRPE binary (default: '-u -m 8192') |
 
-## Troubleshooting
+## Dépannage
 
-Please find the troubleshooting documentation for NRPE checks in the [dedicated chapter](../getting-started/how-to-guides/troubleshooting-plugins.md#nrpe-checks) of the Centreon documentation.
+Veuillez trouver la documentation de dépannage pour les contrôles NRPE dans le [chapitre dédié](../démarrer/guides/troubleshooting-plugins.md#nrpe-checks) de la documentation Centreon.
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md:196]: It's not necessary to translate link here. Use the one from the EN part.

File: pp/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md [hunk 2]
```diff
@@ -11,93 +11,129 @@ import TabItem from '@theme/TabItem';
 This Plugin Pack allow to get metrics and statuses collected thanks to the NSClient++
```
Comment (Sims24) [inline pp/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md:11]:
```suggestion
This Plugin Pack allows to get metrics and statuses collected thanks to the NSClient++ 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md [hunk 3]
```diff
@@ -5,153 +5,184 @@ title: Windows NRPE 0.5
...
 
-This Plugin Pack allow to get metrics and statuses collected thanks to the NSClient++ 
-monitoring agent and its embedded NRPE Server. 
+Ce Plugin Pack permet de récupérer les métriques et les statuts collectés grâce à l'agent de monitoring NSClient++ et son serveur NRPE embarqué. 
 
-## Pack assets
+### Modèles
 
-### Monitored objects
+Le pack de plugins Centreon **Windows NSClient 0.5** apporte 1 modèle d'hôte :
```
Comment (Sims24) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md:15]:
```suggestion
Le pack **Windows NSClient 0.5** apporte 1 modèle d'hôte :
```

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md (hunk 1 above)
Comment (cedricmeschin) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md]: ok, fixed

**Review by Sims24 — APPROVED** [review]

**Pull request merged**